src/ParallelAlgorithms/Interpolation/Events/InterpolateWithoutInterpComponent.hpp

```diff
@@ -77,28 +108,175 @@ class InterpolateWithoutInterpComponent<VolumeDim, InterpolationTargetTag,
       detail::get_compute_items_on_source_or_default_t<InterpolationTargetTag,
                                                        tmpl::list<>>;
 
-  using argument_tags =
-      tmpl::list<typename InterpolationTargetTag::temporal_id,
-                 Tags::InterpPointInfoBase,
-                 ::Events::Tags::ObserverMesh<VolumeDim>, SourceVarTags...>;
+  using argument_tags = tmpl::list<
+      typename InterpolationTargetTag::temporal_id, Tags::InterpPointInfoBase,
+      ::Events::Tags::ObserverMesh<VolumeDim>,
+      ::Events::Tags::ObserverCoordinates<VolumeDim, frame>, SourceVarTags...>;
```

Comment:
Something I just realized might be a problem: In FD we have gaps between grid points, so if one of the target points ends up between grid points of neighboring elements, would this cause it not to interpolate? The solution would probably be to take the DG coords as the bounds, and then interpolate with `ObserverMesh`

Reply:
Yeah I think this would cause it not to call the post interpolation callback because it'd be waiting to receive all the target points and just wouldn't receive one. Is there a tag for only the DG coords that is accessible to non-subcell code?

Reply:
`domain::Tags::Coordinates` always gives the DG coords in all executables :)

Reply:
Oh ok seems simple enough to just swap out `::Events::Tags::ObserveCoordinates` for `domain::Tags::Coordinates`. And the observer mesh is already used for the interpolation so that's taken care of.

Reply:
Yep! You want the `ObserverMesh`, though, since that's what the interpolation code uses :)

Reply:
Is the `::Events::Tags::ObserverMesh<VolumeDim>` that's already there not what we want? I thought it was

Reply:
Yes, sorry, I was trying to say "Don't change that one, only the coordinates"

Reply:
Ok done, squashed, and added a comment